docs/src/main/asciidoc/cdi-integration.adoc

```diff
@@ -279,7 +280,7 @@ void doSomethingWithNamedBeans(SynthesisFinishedBuildItem synthesisFinished, Bui
 Sometimes it is practical to be able to register a _synthetic bean_.
 Bean attributes of a synthetic bean are not derived from a Java class, method or field.
 Instead, all the attributes are defined by an extension.
-In regular CDI, this could be achieved using the https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#after_bean_discovery[`AfterBeanDiscovery.addBean()`, window="_blank"] methods.
+In regular CDI, this could be achieved using the https://jakarta.ee/specifications/cdi/4.0/jakarta-cdi-spec-4.0.html#after_bean_discovery[`AfterBeanDiscovery.addBean()`, window="_blank"] methods.
```

Comment:
Similarly to my comment above, maybe we want to relate this to Lite API rather than Full? Or perhaps mention both :shrug:

Reply:
Done.

Note that we only point to Portable Extensions (and with this PR also Build Compatible Extensions) on 2 places, so we could perhaps just delete these references altogether. (Alternatively, we could point to corresponding PExtensions / BCExtensions APIs on all appropriate places?)

Reply:
We could do that as well.
Maybe it would be more beneficial to note what's the difference between using CDI API and Quarkus API. I.e. the ability to simply tweak just CDI parts versus the ability to perform deep and complex integration with basically any other Quarkus extension?